.github/workflows/test.yml

```
@@ -88,3 +88,55 @@ jobs:
           git add bricks
           git commit -m "[Chore] Generate Mason bundle"
           git push
+
+  generate_sample_project:
+    # The job generate_sample_project identify that the job generate_mason_bundle must complete successfully
+    # before this generate_sample_project job will run
+    needs: generate_mason_bundle
+    name: Generate the sample project
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.WIKI_ACTION_TOKEN }}
+          ref: ${{ github.ref }}
```

I think we need `head_ref` instead of `ref` 🤔 
- When the workflow is a pull_request, head_ref contains the source ref, and ref contains the target branch. 
- When the workflow is a push, ref contains the branch pushed.
https://laurentsenta.com/digital-garden/programming/github-actions-tips/

```suggestion
          ref: ${{ github.head_ref }}
```

---

@nmint8m please check my test on https://github.com/nimblehq/flutter-templates/pull/260/commits/475294982adfb2dbdfd41117078a6c50b1ef927a